nixos/modules/virtualisation/proxmox-image.nix

```diff
@@ -234,6 +253,8 @@ with lib;
         mkdir -p $out/nix-support
         echo "file vma $out/vzdump-qemu-${cfg.filenameSuffix}.vma.zst" >> $out/nix-support/hydra-build-products
       '';
+      bootSize = cfg.qemuConf.bootSize;
+      diskSize = cfg.qemuConf.diskSize;
```

Comment:
```suggestion
      inherit (cfg.qemuConf) diskSize bootSize;
```

Reply:
Sounds great!
Didn't know this would be possible!
What do you think about also adding `additionalSpace` ?
I just realised I could add this option as well.

Reply:
> Yep, that would be useful to have too.

* added it as well
* forced pushed the changes with better fitting commit message
* changed name of PR
* tested the `additionalSpace` option -> worked just fine